Dockerfile

```diff
@@ -16,7 +14,10 @@ RUN zypper -n install \
         python3-CherryPy python2-pecan python3-pecan python2-Jinja2 \
         python3-Jinja2 python2-pyOpenSSL python3-pyOpenSSL \
         python3-Werkzeug python3-bcrypt python3-Routes python3-requests \
-        gcc7 gcc7-c++ libstdc++6-devel-gcc7 python2-PyJWT python3-PyJWT
+        gcc7 gcc7-c++ libstdc++6-devel-gcc7 python2-PyJWT python3-PyJWT \
+        python2-devel python3-devel
+
+RUN zypper -n install --repo "Ceph Nautilus (openSUSE_Tumbleweed)" python3-Cython python2-Cython
```

Comment:
Just wondering: would it make sense to create an alias when adding this repo in line 4, and then refer to this repo by alias instead of this long name with whitespace?

Reply:
Good idea, I will check this and adjust PR if will be working.

Reply:
They alreaedy downgraded the version in Tumbleweed, so I think we don't need to add the extra repository or specify the version.

Reply:
Indeed, for now this workaround does not seem to be necessary anymore.